docs/source/tutorial/tutorial-authentication.mdx

```diff
@@ -1,38 +1,36 @@
 ---
-title: Authentication
+title: 7. Enable authentication
 ---
 
 import LoginUISetupPanel from "./components/login_ui_setup_panel.mdx"
 
-In this section, you'll log in to the server and get a token you can use to make identified requests. 
+In this section, you'll add the ability to log in to your server and obtain a token that your client can use to make identified requests.
```

Comment:
I would probably leave it as `the server` or `the example server` and not `your server` for the reasons outlined below